.github/workflows/msbuild.yml

```diff
@@ -161,7 +161,7 @@ jobs:
           output: both
           # Fail if less than 86% total coverage, measured across all packages combined.
           # Report "yellow" status if less than 90% total coverage.
-          thresholds: '86 90'
+          thresholds: '85 90'
```

Comment:
I don't get it. If you measure an absolute number of uncovered lines, you'll have to update this absolute number for every PR, right? Otherwise, if you only add uncovered lines of code, it would not complain which is not desirable.

Reply:
> I don't get it. If you measure an absolute number of uncovered lines, you'll have to update this absolute number for every PR, right?

If you only add covered lines, then the uncovered line count does not change, so you would not have to update anything.

Reply:
If you only add covered lines, then the coverage percentage is automatically going up, so you wouldn't have to update anything if the threshold stays the same. The covered line percentage is an average, adding anything above this average will increase the average, which thus will remain above the threshold. Am I missing something?